main/webservices/registration.soap.php

```diff
@@ -4922,11 +4991,23 @@ function WSUnsubscribeUserFromCourse($params)
     )
 );
 
+$server->wsdl->addComplexType(
+    'unSubscribeUserToCourseSimple_return',
+    'complexType',
+    'struct',
+    'all',
+    '',
+    array(
+        'original_user_id_value'    => array('name' => 'original_user_id_value',    'type' => 'xsd:string'),
+        'original_course_id_value'  => array('name' => 'original_course_id_value',  'type' => 'xsd:string'),
+        'result'                    => array('name' => 'result',                    'type' => 'xsd:int')
+    )
+);
 
 // Register the method to expose
 $server->register('WSUnSubscribeUserFromCourseSimple',                         // method name
     array('unSubscribeUserFromCourseSimple' => 'tns:unSubscribeUserFromCourseSimple'), // input parameters
-    array('return' => 'tns:result_createUsersPassEncrypt'),           // output parameters
+    array('return' => 'tns:unSubscribeUserToCourseSimple_return'),           // output parameters
```

Comment:
I changed this to return the original_course_id_value as well as the original_user_id_value in the webservice result.